api/examples/list.ts

```diff
@@ -20,15 +20,8 @@ export default withApiHandler(async function(
     return res.send(exampleList);
   }
 
-  await Promise.all([
-    extract('https://github.com/zeit/now/archive/master.zip', '/tmp'),
-    extract('https://github.com/zeit/now-examples/archive/master.zip', '/tmp'),
-  ]);
-
-  const exampleList = new Set([
-    ...summary('/tmp/now-master/examples'),
-    ...summary('/tmp/now-examples-master'),
-  ]);
+  await extract('https://github.com/zeit/now/archive/master.zip', '/tmp');
+  const exampleList = summary('/tmp/now-master/examples');
```

Comment:
Can we change this since we have a monorepo?

Thinking out loud.....could the files be compressed as part of the build script? Then there is no need to fetch from github each time? 🤔

Reply:
The results are already cached for 24 hours on the edge

Reply:
We could cache forever with immutable header if we had all of the file available at build time. Because this is simply ready from master which is what already happens with the github integration 💡

Reply:
And if we'd made it at build time we'd make the lambdas larger and increase the build time

Reply:
Perhaps each example could be its own lambda. Surely they are all under 50MB 🤔